snap/snapcraft.yaml

```diff
@@ -8,8 +8,63 @@ description: |
   store.
 
 confinement: devmode # use 'strict' once you have the right plugs and slots
+grade: devel
+icon: snap/gui/icon.png
+
+apps:
+  my-app:
+    command: my-app
+    desktop: share/applications/my-app.desktop
+    #plugs:
+    #- desktop
+    #- desktop-legacy
+    #- x11
+    #- unity7
+
+    #slots:
 
 parts:
-  my-part:
+  # Patches to fix other parts
+  patches:
+    source: snap/patches
+    source-type: local
+    plugin: dump
+
+    # DISABLED: Bug #1775582 “`organize:{ /: another-dir/ }` causes the items under host root directory to be copied in another-dir” : Bugs : Snapcraft
+    # https://forum.snapcraft.io/t/organize-another-dir-causes-the-items-under-host-root-directory-to-be-copied-in-another-dir/5806
+    #organize:
+        #/: patches/
+    override-build: |
+      set -eu
+
+      snapcraftctl build
+
+      mkdir \
+        --parents \
+        "$SNAPCRAFT_PART_INSTALL"/patches
+      mv \
+        "$SNAPCRAFT_PART_INSTALL"/*.diff \
+        "$SNAPCRAFT_PART_INSTALL"/*.patch \
+        "$SNAPCRAFT_PART_INSTALL"/*.sed \
+        "$SNAPCRAFT_PART_INSTALL"/patches \
+        || true # Empty patches folder is allowed
```

Comment:
this override script isn't needed, please drop.

Reply:
I really don't think we should be complicating a template yaml with patches. They are only required in rare cases.